README.md

```diff
@@ -38,6 +38,10 @@ providers:
     class: octodns_etchosts.EtcHostsProvider
     # The output directory for the hosts file <zone>.hosts
     directory: ./hosts
+    # Remove trailing dots of zone names (e.g. example.com. => example.com)
+    # Avoids problems with certain DNS providers, as the host file format requires an alphanumeric character to be the final character in a hostname.
+    # Default: False
+    remove_trailing_dots: False
```

Comment:
Should this default to `true`? I realize it's a change in behavior, but if the spec says no dot then it's probably a preferable default. Interesting that it didn't cause any issues when I tested it out in the past on OSX and Linux.

Reply:
Since it's a change in behavior, I've refrained from making it default to `true`. But yes, it would be more suiting as the spec requires it.
Should I change it to default: `true` and update the test cases?

Reply:
Since the spec is so clear I'd consider it a worthwhile change of the default behavior and I can't think of a situation where it would break anything.